pkg/proxy/util/utils.go

```diff
@@ -156,6 +156,21 @@ func GetLocalAddrs() ([]net.IP, error) {
 	return localAddrs, nil
 }
 
+// GetLocalAddrSet return a local IPSet.
+// If failed to get local addr, will assume no local ips.
+func GetLocalAddrSet() utilnet.IPSet {
+	localAddrs, err := GetLocalAddrs()
+	if err != nil {
```

Comment:
I'm torn.  Part of me feels like this should be logged from the caller, which reduces this function to just a few LOC.  I guess it doesn't matter much, its an internal API..